### PR TITLE
Added storage updates

### PR DESCRIPTION
**Pull request by rishabpoh**
Added latest release info for storage libraries

### CODE REVIEW

File: releases/2020-02/java.md [hunk 1]
```diff
@@ -74,37 +74,37 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-blob</artifactId>
-  <version>12.2.0</version>
+  <version>12.4.0-beta.1</version>
```
Comment (alzimmermsft) [inline releases/2020-02/java.md]: Versions appear to be incorrect

Comment (gapra-msft) [reply releases/2020-02/java.md]: 12.4.0

File: releases/2020-02/java.md [hunk 2]
```diff
@@ -74,37 +74,37 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
   <artifactId>azure-storage-blob-batch</artifactId>
-  <version>12.2.0</version>
+  <version>12.4.0-beta.1</version>
 </dependency>
 
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-blob-cryptography</artifactId>
-  <version>12.2.0</version>
+  <version>12.4.0-beta.1</version>
```
Comment (gapra-msft) [inline releases/2020-02/java.md]: 12.4.0

File: releases/2020-02/java.md [hunk 3]
```diff
@@ -74,37 +74,37 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
   <artifactId>azure-storage-file-share</artifactId>
-  <version>12.1.0</version>
+  <version>12.2.0-beta.1</version>
 </dependency>
 
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-queue</artifactId>
-  <version>12.2.0</version>
+  <version>12.3.0-beta.1</version>
```
Comment (gapra-msft) [inline releases/2020-02/java.md]: 12.3.0

File: releases/2020-02/java.md [hunk 4]
```diff
@@ -74,37 +74,37 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
   <artifactId>azure-storage-blob-cryptography</artifactId>
-  <version>12.2.0</version>
+  <version>12.4.0-beta.1</version>
 </dependency>
 
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-file-datalake</artifactId>
-  <version>12.0.0-beta.9</version>
+  <version>12.0.0-beta.11</version>
```
Comment (gapra-msft) [inline releases/2020-02/java.md]: beta 12

File: releases/2020-02/java.md [hunk 5]
```diff
@@ -74,37 +74,37 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
 </dependency>
 
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-file-share</artifactId>
-  <version>12.1.0</version>
+  <version>12.2.0-beta.1</version>
```
Comment (gapra-msft) [inline releases/2020-02/java.md]: 12.2.0

**Pull request merged**